app/database/server/schema/table_schemas/channel.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {tableSchema} from '@nozbe/watermelondb';
+
+import {MM_TABLES} from '@constants/database';
+
+const {CHANNEL} = MM_TABLES.SERVER;
+
+export default tableSchema({
+    name: CHANNEL,
+    columns: [
+        {name: 'create_at', type: 'number'},
+        {name: 'creator_id', type: 'string', isIndexed: true},
+        {name: 'delete_at', type: 'number'},
+        {name: 'display_name', type: 'string'},
+        {name: 'is_group_constrained', type: 'boolean'},
```

Comment:
I've approved; but the docs have `IsGroupContrained` (misspelled constrained).

Reply:
we should fix the doc